session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java

```diff
@@ -148,7 +148,9 @@ public void sessionTouched(SessionData session) {
     public void run() {
       long prevMin = roundDownMinute(System.currentTimeMillis());
 
-      logger.info("Cleaning up sessions expiring at {}", prevMin);
+      if (logger.isDebugEnabled()) {
```

Comment:
No need for this mask as logger.debug will already do it and there are no calculations in parameters. Same applies to next few calls.